alf/utils/common.py

```diff
@@ -295,11 +292,11 @@ def cast_transformer(observation, dtype=tf.float32):
     """
 
     def _cast(obs):
-        if isinstance(obs, tf.Tensor):
-            return tf.cast(obs, dtype)
+        if isinstance(obs, torch.Tensor):
+            return torch.type(obs, dtype)
```

Comment:
```torch.type``` does not exist?

Reply:
torch.type does exist

Reply:
I think you need `obs.type(dtype)`. torch.type doesn't exist.

Reply:
Fixed